src/DynamoCoreWpf/ViewModels/Core/OutPortViewModel.cs

```diff
@@ -14,6 +16,11 @@ public partial class OutPortViewModel : PortViewModel
         private DelegateCommand hideConnectionsCommand;
         private DelegateCommand portMouseLeftButtonOnContextCommand;
 
+        private SolidColorBrush portValueMarkerColor = new SolidColorBrush(Color.FromArgb(255, 204, 204, 204));
+        private static SolidColorBrush PortValueMarkerBlue = new SolidColorBrush(Color.FromRgb(106, 192, 231));
+        private static SolidColorBrush PortValueMarkerRed = new SolidColorBrush(Color.FromRgb(235, 85, 85));
+        private static SolidColorBrush PortValueMarkerGrey = new SolidColorBrush(Color.FromRgb(153, 153, 153));
```

Comment:
I think we ran into similar case before but still curious, any chance we can define these colors in style XML?

Reply:
Yes, that's possible! Should we define the colors of the input port in the XML as well?

Reply:
Actually we did eventually solve the issue with defining colors in code (it was a unit testing issue). If the unit tests load the view, which I think was a flag of some kind, the ResourceDictionaries can be referenced without causing an Exception. In short, we can move color references to the relevant ResourceDictionary and keep everything centralised.

Reply:
Ok, I will set the colors in DynamoColorsAndBrushes.xml. Thanks!